benchmarks/benchmarks/test_go_benchmark_functions.py

```diff
@@ -40,6 +40,17 @@ def test_solution_exists(self):
             # should result in an attribute error if it doesn't exist
             val = f.fglob
 
+    def test_bounds_access_subscriptable(self):
+        # In Python 2 zip returns a list which is subscriptable
+        # In Python 3 zip returns a zip object, which is not subscriptable
+        for name, klass in self.benchmark_functions:
+            if (name in ['Benchmark', 'LennardJones'] or
+                    name.startswith('Problem')):
```

Comment:
A comment about what's different about ``LennardJones`` would be helpful here, I don't see it so quickly.